### PR TITLE
adding aarch64 + ppc64le

### PR DESCRIPTION
**Pull request by ngam**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
@conda-forge/core could you please merge? thanks

File: conda-forge.yml [hunk 1]
```diff
@@ -1 +1,6 @@
 conda_forge_output_validation: true
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
+os_version:
+  linux_64: cos7
```
Comment (xhochy) [inline conda-forge.yml]: As in the other PR: Why?

Comment (ngam) [reply conda-forge.yml]: Copy-paste 😵‍💫

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
Thanks @xhochy for spotting that. Please merge when you have a chance. Sorry about the mistake.

**Pull request merged**